scripts/qa_crowbarsetup.sh

```diff
@@ -3273,14 +3280,21 @@ function onadmin_run_cct()
     # - install cct dependencies
     ensure_packages_installed git-core gcc make ruby2.1-devel
 
-    if iscloudver 6plus && [[ -n $cct_tests ]]; then
+    if [ -n $cct_tests ]; then
 
         # - default checkout location for mkcloud based setups:
         mkdir -p /root/github.com/SUSE-Cloud
         pushd /root/github.com/SUSE-Cloud/
         git clone https://github.com/SUSE-Cloud/cct.git
-        # TODO checkout branches for older versions
         cd cct
+
+        # checkout branches if needed
+        case "$cloudsource" in
+            develcloud5|GM5|GM5+up)
+                git checkout "cloud5"
```

Comment:
please note you can also pass the branchname to "git clone" directly with -b <branch>.